stracker/items/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from core.models import BaseAbstractModel
+from items import enums
+
+
+class Items(BaseAbstractModel):
+    """
+    Class based Items model
+    type is Charfield with choices comes from enums.py
+    """
+    name = models.CharField(_("name"), max_length=150, null=False, blank=False)
+    type = status = models.CharField(
```

Comment:
type builtin function ile çalıkışıyor ismi, farklı bir isim kullanmak daha iyi olur. Ayrıca type ve status yanlışlıkla eşitlenmiş sanırım
```suggestion
    itype = models.CharField(
```

Reply:
Sanırım orada bir miss click olmuş. En başta migration yaptığımda sorun çıkmamıştı. Farketmemişim. Type ismini de item_type olarak değiştirmeyi düşünüyorum.